conf/eslint.json

```diff
@@ -116,6 +116,7 @@
         "no-useless-call": "off",
         "no-useless-concat": "off",
         "no-useless-constructor": "off",
+        "no-useless-escape": "off",
```

Comment:
This is the setting for `extends: "eslint:recommended"`, so we have to avoid a breaking change until v3.0.0.

But I think good to use this rule in `eslint-config-eslint`: https://github.com/eslint/eslint/blob/master/packages/eslint-config-eslint/default.yml